recipes/TextFieldsHalogen/src/Main.purs

```diff
@@ -0,0 +1,31 @@
+module HelloHalogen.Main where
+
+import Prelude
+
+import Data.Maybe (Maybe(..))
+import Effect (Effect)
+import Halogen as H
+import Halogen.Aff as HA
+import Halogen.HTML as HH
+import Halogen.HTML.Events as HE
+import Halogen.HTML.Properties as HP
+import Halogen.Hooks as Hooks
+import Halogen.VDom.Driver (runUI)
+
+main :: Effect Unit
+main =
+  HA.runHalogenAff do
+    body <- HA.awaitBody
+    void $ runUI hookComponent Nothing body
+
+hookComponent
+  :: forall unusedQuery unusedInput unusedOutput anyMonad
+   . H.Component HH.HTML unusedQuery unusedInput unusedOutput anyMonad
+hookComponent = Hooks.component \_ _ -> Hooks.do
+  state /\ stateIdx <- useState ""
+  Hooks.pure $
+    HH.div_
+      [ HH.input
+        [ HP.placeholder "Text to reverse"
+        , HP.value state
+        , HE.onInput \newValue -> Just $ Hooks.put stateIdx (?reverseString newValue)
```

Comment:
Somehow, CI built find despite this line. `?reverseString` should have created a compiler error, making the build fail. However, I don't see any such errors [in the build for this recipe](https://travis-ci.com/github/JordanMartinez/purescript-cookbook/builds/173738445#L421). That's a problem...

Reply:
This issue was actually caused by the `web/index.js` file referring to the original recipe I duplicated when I originally started working on this recipe